disc/cog.py

```diff
@@ -19,7 +19,25 @@ def __init__(self, bot):
     async def _post_job(self, ctx: SlashContext, job):
         linkedin = LinkedIn()
         data = linkedin.get_job_info(job)
-        await ctx.send(f"Title: {data.title}")
+
+        colors = Colors()
+        random_color = colors.random_color()
+        final_color = discord.Colour(random_color)
+
+        description = f"**{data.company}**\n*{data.title}*\n\n"
+        description += f"Location: {data.location}.\n\n"
+        description += f"This job was posted __{data.posted_time_ago}__.\n\n"
+        description += f"{data.summary}\n\nApply Now: {data.url}\n"
```

Comment:
@truongluan303 I know this looks a little crazy but I changed the description implementation to use string concatenation due to an indentation bug found on mobile devices. See photo below:
![IMG_3815](https://user-images.githubusercontent.com/63386979/183255781-aa5ab209-8e95-414e-b644-38d2e9d9c9d2.PNG)

With the string concatenation implementation, the bug fix now looks like this:
![IMG_3816](https://user-images.githubusercontent.com/63386979/183255804-bde4cade-e17c-4825-a367-86e637fcd0bd.PNG)

Let me know if you have any feedback on this or suggestions.

Reply:
We have 2 options to fix this.
You can either line up with the `"""`, like this:
```python
string = """line one
            line two
            line three"""
```
However, IMO, the cleanest way is:
```python
string = (
    "line 1\n"
    "line 2\n"
    "line 3\n"
)
Notice we have no commas
```